src/main/java/bio/terra/service/filedata/flight/ingest/IngestFileAzurePrimaryDataLocationStep.java

```diff
@@ -36,21 +33,12 @@ public StepResult doSkippableStep(FlightContext context) throws InterruptedExcep
     FlightMap workingMap = context.getWorkingMap();
     Boolean loadComplete = workingMap.get(FileMapKeys.LOAD_COMPLETED, Boolean.class);
     if (loadComplete == null || !loadComplete) {
-      // Retrieve the already authorized billing profile from the working map and retrieve
-      // or create a storage account in the context of that profile and the dataset.
-      BillingProfileModel billingProfile =
-          workingMap.get(ProfileMapKeys.PROFILE_MODEL, BillingProfileModel.class);
-
       try {
-        AzureStorageAccountResource storageAccountResource =
-            resourceService.getOrCreateDatasetStorageAccount(
-                dataset, billingProfile, context.getFlightId());
-        workingMap.put(FileMapKeys.STORAGE_ACCOUNT_INFO, storageAccountResource);
-        AzureStorageAuthInfo storageAuthInfo =
-            AzureStorageAuthInfo.azureStorageAuthInfoBuilder(
-                billingProfile, storageAccountResource);
-        workingMap.put(FileMapKeys.STORAGE_AUTH_INFO, storageAuthInfo);
+        // Retrieve the already authorized billing profile from the working map and retrieve
+        // or create a storage account in the context of that profile and the dataset.
+        getOrCreateDatasetStorageAccount(context);
 
+        // TODO - is this just left over from GCP? I don't expect a bucket lock exception here
       } catch (BucketLockException blEx) {
         return new StepResult(StepStatus.STEP_RESULT_FAILURE_RETRY, blEx);
       }
```

Comment:
I'm going to remove this unless anyone knows why we'd need to catch a bucket lock exception on creating an azure storage account.